CMSgov/mint-app
# [EASI-2313] Email Templating / Collaborator Added Email

# EASI-2312

## Changes and Description

- Added a shared package for email templates
- Added mock unit testing for collaborator added emails
- Added email sending on collaborator added

To Do:
_Normally I wouldn't promote a PR while there are known issues but this one is large and I want people to be able to review it while I make last second fixes. Those can then be parsed in the specific commit to ease the burden of review._
- ~~Fix a bug with the HostName directive being malformed~~
- ~~Work with FrontEnd to create a URL alias for unfollowing (or at least make a ticket)~~ @patrickseguraoddball to the rescue!



## How to test this change

- Add a collaborator to a model and observe that MailCatcher captures associated email(s)
- Run the TestCreatePlanCollaborator unit test, observe that it succeeds
- Run the TestAddedAsCollaboratorEmail unit test, observe that it succeeds



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
